package.json

```
@@ -25,7 +25,7 @@
     "test:watch": "jest app/javascript/ bin/ --watch",
     "postcss": "postcss public/assets/*.css -d public/assets 2> postcss_error.log",
     "e2e": "bin/e2e",
-    "e2e:noseed": "yarn run e2e --no-seed"
+    "e2e:minimal-seed": "yarn run e2e --minimal-seed"
```

Nitpick (non-blocking): Consider a shorter command name.

```suggestion
    "e2e:minseed": "yarn run e2e --minimal-seed"
```

---

@nickytonline and this one `e2e:onboardingseed`?

---

I'm not sure what we can shorten this to, maybe `onbseed`, although this isnt very semantic so I would prefer the fullname or even one with the `-`.

---

Yeah, now that we'll be calling it admin onboarding for the flows folder, go ahead with a longer name to make it clear.